app/ios/App/Podfile

```diff
@@ -10,7 +10,7 @@ def capacitor_pods
   # Automatic Capacitor Pod dependencies, do not delete
   pod 'Capacitor', :path => '../../node_modules/@capacitor/ios'
   pod 'CapacitorCordova', :path => '../../node_modules/@capacitor/ios'
-  pod 'CapacitorCommunityIntercom', :path => '..\..\node_modules\@capacitor-community\intercom'
+  pod 'CapacitorCommunityIntercom', :path => '../../node_modules/@capacitor-community/intercom'
```

Comment:
was this the real issue? It was like targeting Windows before, wasn't it?

Reply:
no this is not the issue. jordan added another appid to the associated apps